rules/integrations/cloud_defend/persistence_ssh_authorized_keys_modification_inside_a_container.toml

```diff
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2023/05/12"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "New Integration: Cloud Defend"
+min_stack_version = "8.8.0"
+updated_date = "2023/05/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the creation or modification of an authorized_keys or sshd_config file inside a container. 
+The Secure Shell (SSH) authorized_keys file specifies which users are allowed to log into a server using public key authentication. 
+Adversaries may modify it to maintain persistence on a victim host by adding their own public key(s). 
+Unexpected and unauthorized SSH usage inside a container can be an indicator of compromise and should be investigated. 
+"""
+from = "now-6m"
+index = ["logs-cloud_defend*"]
+interval = "5m"
+language = "eql"
+license = "Elastic License v2"
+name = "SSH Authorized Keys File Modified Inside a Container"
+risk_score = 73
+rule_id = "f7769104-e8f9-4931-94a2-68fc04eadec3"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Persistence", "Lateral Movement", "Container"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+file where container.id:"*" and
+  event.type in ("change", "creation") and 
+  event.action in ("creation", "modification") and 
```

Comment:
I'm not sure you need both event.type and event.action here. It's redundant as they both specify the same activity. I think could choose one or the other and get the same result.